### PR TITLE
Fix Stash addons name + Remove addon installation requirements

### PR DESCRIPTION
**Pull request by hossainemruz**
Signed-off-by: Emruz Hossain <emruz@appscode.com>

**Tasks:**
- [x] Elasticsearch
- [x] MariaDB
- [x] MongoDB
- [x] MySQL
- [x] Percona XtraDB
- [x] PostgreSQL

### CODE REVIEW

File: docs/guides/elasticsearch/backup/auto-backup/index.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ In this tutorial, we are going to show how you can configure a backup blueprint
 ## Before You Begin
 
 - At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster.
-- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
 - Install KubeDB in your cluster following the steps [here](/docs/setup/README.md).
+- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
```
Comment (spectro30) [inline docs/guides/elasticsearch/backup/auto-backup/index.md]: *Enterprise

File: docs/guides/elasticsearch/backup/kubedb/index.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ Stash 0.9.0+ supports backup and restoration of Elasticsearch clusters. This gui
 ## Before You Begin
 
 - At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster.
-- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
 - Install KubeDB in your cluster following the steps [here](/docs/setup/README.md).
+- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
```
Comment (spectro30) [inline docs/guides/elasticsearch/backup/kubedb/index.md]: *Enterprise

File: docs/guides/mariadb/backup/logical/cluster/index.md [hunk 3]
```diff
@@ -18,9 +18,8 @@ Stash `v0.11.8+` supports backup and restoration of MariaDB databases. This guid
 ## Before You Begin
 
 - At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster.
-- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
-- Install MariaDB addon for Stash following the steps [here](https://stash.run/docs/latest/addons/mariadb/setup/install/).
 - Install KubeDB operator in your cluster from [here](https://kubedb.com/docs/latest/setup).
+- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
```
Comment (spectro30) [inline docs/guides/mariadb/backup/logical/cluster/index.md]: *Enterprise

File: docs/guides/mariadb/backup/logical/standalone/index.md [hunk 4]
```diff
@@ -18,9 +18,8 @@ Stash `v0.11.8+` supports backup and restoration of MariaDB databases. This guid
 ## Before You Begin
 
 - At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster.
-- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
-- Install MariaDB addon for Stash following the steps [here](https://stash.run/docs/latest/addons/mariadb/setup/install/).
 - Install KubeDB operator in your cluster from [here](https://kubedb.com/docs/latest/setup).
+- Install Stash Enterise in your cluster following the steps [here](https://stash.run/docs/latest/setup/install/enterprise/).
```
Comment (spectro30) [inline docs/guides/mariadb/backup/logical/standalone/index.md]: *Enterprise

File: docs/guides/postgres/backup/standalone/index.md [hunk 5]
```diff
@@ -73,7 +72,7 @@ spec:
 Create the above `Postgres` crd,
 
 ```bash
-$ kubectl apply -f https://github.com/standalone/docs/raw/{{< param "info.version" >}}/docs/guides/postgres/backup/standalone/examples/backup/postgres.yaml
+$ kubectl apply -f https://github.com/standalone/docs/raw/{{< param "info.version" >}}/docs/guides/postgres/backup/standalone/examples/postgres.yaml
```
Comment (tamalsaha) [inline docs/guides/postgres/backup/standalone/index.md:75]: This link is not valid.

Comment (hossainemruz) [reply docs/guides/postgres/backup/standalone/index.md]: Fixed

**Pull request merged**